frontend/website/src/Testnet.re

```diff
@@ -0,0 +1,179 @@
+      media(Style.MediaQuery.somewhatLarge, [flexDirection(`row)]),
+    ]);
+
+  let leaderboard =
+    style([
+      background(Style.Colors.hyperlinkAlpha(0.15)),
+      width(`percent(100.)),
+      maxWidth(rem(41.)),
+      borderRadius(px(3)),
+      padding(rem(1.)),
+      Style.Typeface.pragmataPro,
+      lineHeight(rem(1.5)),
+      color(Style.Colors.midnight),
+      selector(
+        ".leaderboard-row",
+        [
+          display(`grid),
+          gridColumnGap(rem(1.)),
+          gridTemplateColumns([rem(2.5), `auto, rem(6.)]),
+        ],
+      ),
+      selector("div span:nth-child(odd)", [justifySelf(`flexEnd)]),
+      selector(
+        "hr",
+        [
+          height(px(4)),
+          borderTop(px(1), `dashed, Style.Colors.marine),
+          borderLeft(`zero, solid, transparent),
+          borderBottom(px(1), `dashed, Style.Colors.marine),
+        ],
+      ),
+      selector(
+        "#leaderboard-loading",
+        [
+          textAlign(`center),
+          marginTop(rem(2.)),
+          color(Style.Colors.slateAlpha(0.7)),
+        ],
+      ),
+    ]);
+
+  let headerRow =
+    merge([
+      Style.Body.basic_semibold,
+      style([
+        display(`grid),
+        color(Style.Colors.midnight),
+        gridColumnGap(rem(1.)),
+        gridTemplateColumns([rem(2.5), `auto, rem(6.)]),
+      ]),
+    ]);
+
+  let copy =
+    style([
+      width(rem(28.)),
+      margin3(~top=`zero, ~h=`auto, ~bottom=rem(2.)),
+      media(Style.MediaQuery.somewhatLarge, [marginLeft(rem(5.))]),
+      ...Style.Body.basicStyles,
+    ]);
+};
+
+let component = ReasonReact.statelessComponent("Testnet");
+let make = _children => {
+  ...component,
+  render: _self => {
+    <div>
+      <div className=Styles.header>
+        <h1 className=Style.H1.hero>
+          {ReasonReact.string("Testnet Leaderboard")}
+        </h1>
+      </div>
+      <div className=Styles.content>
+        <div id="testnet-leaderboard" className=Styles.leaderboard>
+          <div className=Styles.headerRow>
+            <span> {ReasonReact.string("Rank")} </span>
+            <span> {ReasonReact.string("Username")} </span>
+            <span> {ReasonReact.string("Total Points")} </span>
+          </div>
+          <hr />
+          <div id="leaderboard-loading">
+            {ReasonReact.string("Loading...")}
+          </div>
+        </div>
+        <div className=Styles.copy>
+          <p>
+            {ReasonReact.string(
+               "The goal of Testnet Points* is to recognize Coda community members who are actively involved in the network. There will be regular challenges to make it fun, interesting, and foster some friendly competition! Points can be won in several ways like being first to complete a challenge, contributing code to Coda, or being an excellent community member and helping others out.",
+             )}
+          </p>
+          <p> <strong> {ReasonReact.string("Challenge #1")} </strong> </p>
```

Comment:
qq - would it help to have these in the google sheet so instead of updating code every time, we can just update there and it would grab the new challenges? Also Christine or Claire would be able to update it that way

Reply:
I think this is a great idea for the next iteration. We need to pull a lot more data from the spreadsheets so we should take the challenge copy into account when we add support for seeing scores week by week.